testDefinition/examplesTS/windowChain.ts

```diff
@@ -12,6 +12,7 @@ async function windowTest() {
 	await window().goForward();
 	await window().dismissModal();
 	await window().acceptModal();
```

Comment:
According to validation - acceptModal can't be called without arguments and may fall in runtime with validation error, please check it.

Reply:
sorry, just going back to this.
Do you mean in the `validateStringOrNull = (val, name)` function or somewhere else?

Reply:
No, just need update type definition of this function